usb/usbhost.h

```diff
@@ -64,6 +64,7 @@ typedef struct usb_transfer {
 	unsigned long rid;
 	unsigned int port;
 	pid_t pid;
+	msg_t msg;
```

Comment:
`pid` field became redundant. Please note that `msg_t` is quite hefty (150 bytes)

Reply:
True, this was only a temporary solution to make synchronous transfers work before doing a rework - I could have added a comment about this. A final more thought through solution is in https://github.com/phoenix-rtos/phoenix-rtos-usb/pull/35